ci/build/release-prep.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Description: This is a script to make the release process easier
+# Run it with `yarn release:prep` and it will do the following:
+# 1. Check that you have a $GITHUB_TOKEN set and hub installed
+# 2. Update the version of code-server (package.json, docs, etc.)
+# 3. Update the code coverage badge in the README
+# 4. Open a draft PR using the release_template.md and view in browser
+# If you want to perform a dry run of this script run DRY_RUN=0 yarn release:prep
+
+set -euo pipefail
+
+main() {
+  if [ "${DRY_RUN-1}" -eq 0 ]; then
+    echo "Performing a dry run..."
+    # Disabled because shellcheck wants me to do this
+    # SC2209: Use var=$(command) to assign output (or quote to assign string).
+    # But neither of those match my needs
+    # We do this so that the dry run echos the commands instead of running them
+    # shellcheck disable=SC2209
+    CMD=echo
```

Comment:
Quotes should work! (I think.)

```suggestion
    CMD="echo"
```

Reply:
ooo i didn't even think to try! okay i'll test to verify. good call! (then I can remove shellcheck supression)

Reply:
it worked! 🎉